src/dialect/onnx/onnx_ops.cpp

```diff
@@ -934,6 +956,11 @@ void ONNXConvNoBiasOp::inferShapes() {
 
   // Group is a required attribute and should have default value of 1.
   int64_t group = ONNXConvNoBiasOp::group().getSExtValue();
+
```

Comment:
This is not really necessary; but if you want this, we could push a change to MLIR to record default values

Reply:
This is required for now because the default value is actually not recorded as a physical attribute in the source code so if you attempt to get the attribute it will be null.

Reply:
yes, but that is because using groupAttr() is the wrong interface. You should only be using group(), which builds the default when not specified. So remove this code below and use group(), just as you did on line 958... which is exactly the point!

Reply:
The code that fails is not here it's in the SplitConv rewrite rule. There we actually need the attribute.

Reply:
I see, the issue is with the create op

Reply:
Let's leave it at that, but we should consider a patch to MLIR to save the default attributes